src/index.ts

```diff
@@ -109,7 +109,11 @@ async function serverChannel(
 
   channel.on(START_BUILD, async () => {
     const { projectToken } = projectInfoState.value || {};
-    await runChromaticBuild(localBuildProgress, { projectToken });
+    try {
+      await runChromaticBuild(localBuildProgress, { projectToken });
+    } catch (e) {
+      console.error(`Failed to run Chromatic build, with error:\n${e}`);
```

Comment:
The log could be slightly redundant, but it does report different information than the cli does, and doesn't take up much additional space. I felt like it could be useful if the runChromaticBuild method were to fail for reasons the cli doesn't directly handle well.

If reviewer feels this is unnecessary/noisy I'm open to removing the extra log statement for now.
Example log of when the storybook-build command doesn't exist:
<img width="877" alt="Screen Shot 2023-10-13 at 2 20 44 PM" src="https://github.com/chromaui/addon-visual-tests/assets/5882088/d60e42ff-ee91-4e8e-9e4c-685e688564c5">


```suggestion
  // Output handled by CLI
```